### PR TITLE
Add Plastik token

### PR DESCRIPTION
**Pull request by garciapd**
Add Plastik token!

### CODE REVIEW

File: src/data/alfajores/tokens-info.json [hunk 1]
```diff
@@ -214,5 +214,12 @@
     "imageUrl": "https://raw.githubusercontent.com/valora-inc/address-metadata/main/assets/tokens/ETHIX.png",
     "name": "Ethix",
     "symbol": "ETHIX"
+  },
+  {
+    "address": "0x55067df868997BaD76e831F4bad5B48fbbf4c8D1",
```
Comment (MuckT) [inline src/data/alfajores/tokens-info.json:219]: Please lower-case this address.

File: src/data/mainnet/tokens-info.json [hunk 2]
```diff
@@ -407,5 +407,12 @@
     "imageUrl": "https://raw.githubusercontent.com/valora-inc/address-metadata/main/assets/tokens/ETHIX.png",
     "name": "Ethix",
     "symbol": "ETHIX"
+  },
+  {
+    "address": "0xf4361a8Fbcf6b6A2402f1A1d6c4b945e59a29934",
```
Comment (MuckT) [inline src/data/mainnet/tokens-info.json:412]: Please lower-case this address.

**Review by MuckT — CHANGES_REQUESTED** [review]
The addresses look good in explorer, they just need to be lower cased.

File: src/data/alfajores/tokens-info.json (hunk 1 above)
Comment (garciapd) [reply src/data/alfajores/tokens-info.json]: done!

File: src/data/mainnet/tokens-info.json (hunk 2 above)
Comment (garciapd) [reply src/data/mainnet/tokens-info.json]: done!

**Comment by garciapd** [comment]
@MuckT could you please review?

**Review by MuckT — APPROVED** [review]
[PKS Test Token](https://alfajores-blockscout.celo-testnet.org/token/0x55067df868997BaD76e831F4bad5B48fbbf4c8D1/token-transfers)
[PLASTIK Token (PLASTIK)](https://explorer.celo.org/address/0xf4361a8Fbcf6b6A2402f1A1d6c4b945e59a29934/logs)

**Comment by garciapd** [comment]
@MuckT is 0x55067df868997BaD76e831F4bad5B48fbbf4c8D1

yes

**Pull request merged**